src/bdh.cpp

```diff
@@ -1,9 +1,8 @@
-// -*- mode: C++; c-indent-level: 4; c-basic-offset: 4; indent-tabs-mode: nil; -*-
 //
 //  bdh.cpp -- "Bloomberg Data History" query function for the BLP API
 //
-//  Copyright (C) 2013  Whit Armstrong
-//  Copyright (C) 2015  Whit Armstrong and Dirk Eddelbuettel
+//  Copyright (C) 2013      Whit Armstrong
+//  Copyright (C) 2015-204  Whit Armstrong and Dirk Eddelbuettel
```

Comment:
typo

Reply:
Yup.  Will fix.

Done in https://github.com/Rblp/Rblpapi/pull/391/commits/b89177f8dda809bfb6d17287c6e186186513eed6